app/com/m3/octoparts/aggregator/handler/HttpPartRequestHandler.scala

```diff
@@ -53,11 +53,17 @@ trait HttpPartRequestHandler extends Handler {
    * @return Future[PartResponse]
    */
   def process(partRequestInfo: PartRequestInfo, hArgs: HandlerArguments): Future[PartResponse] = {
-    hystrixExecutor.future {
-      createBlockingHttpRetrieve(partRequestInfo, hArgs).retrieve()
-    }.map {
-      createPartResponse
-    }
+    hystrixExecutor.future(
+      createBlockingHttpRetrieve(partRequestInfo, hArgs).retrieve().copy(),
+      maybeContents => HttpResponse(
+        status = 203, // 203 -> Non-authoritative info
```

Comment:
looking forward to see the face of people who always check status == 200. (don't change anything)